### DISCUSSION TITLE
[Feature Request] A new dynamic recompiler

### DISCUSSION THREAD

**Discussion by Fubuki0x10DE** [86Box/86Box#4385] [General]
https://github.com/86Box/86Box/blob/master/src/codegen/codegen_ops_x86-64.h
A new dynamic recompiler that utilizing SSE2/SSE3 on host processor, and request host processor has SSE2 at least. A 2000 Pentium 4 or 2003 Athlon 64 host processor can support SSE2.

**Comment by TC1995** [comment]
Why insist?
